augur/translate.py

```diff
@@ -114,7 +124,7 @@ def translate_feature(aln, feature):
     '''
     translations = {}
     for sname, seq in aln.items():
-        aa_seq = safe_translate(str(feature.extract(seq)))
+        aa_seq = safe_translate(str(feature.extract(seq)), pad=True)
```

Comment:
When does this happen? I.e. when is an extracted feature from an alignment not the length of the feature?

Reply:
Good point – probably never under typical usage*. I was focusing on changing behavior for calls to reference sequence only, but we could instead remove the padding option entirely.

\* Maybe if `--ancestral-sequences` was aligned to a different reference? Though padding wouldn't be the right way to handle that either.